corehq/apps/reports/standard/cases/case_data.py

```diff
@@ -333,6 +334,7 @@ def form_to_json(domain, form, timezone):
             "username": form.metadata.username if form.metadata else '',
         },
         'readable_name': form_name,
+        'submitter_type': get_submitter_type(form.metadata, domain) if form.metadata else 'Unknown',
```

Comment:
I have just added a question on the ticket whether this should be called `User Type` instead. Otherwise the PR is LGTM.

Reply:
Thank you, if it'll keep things more consistent then it'll be a good idea to change the naming. I'll wait to hear what Mary says on the ticket, and then make the necessary changes if applicable.